nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-remote-process-group.js

```diff
@@ -741,8 +740,8 @@
             })
             .attr('font-family', function (d) {
                 var family = '';
-                if (d.permissions.canRead) {
-                    if (hasIssues(d) || d.component.transmitting) {
+                if (d.permissions.canRead || d.operatePermissions.canWrite) {
```

Comment:
These checks (here and below) also need to consider if the user has `d.permission.canWrite`. If the user only has operate permissions they can see the status. But if the user does not have operate but does have write to the RPG they currently cannot see the status icon.

Reply:
Actually, for determining which icon we show and the appropriate style it appears that we are now only using the status. This is available to everyone so I don't think we need any permission checks here. We should only need them when adding the tooltips. This should be very similar to the Processor Invalid icon and the tooltips for the validation errors.

Reply:
That's true. I've removed permission checks for status icon.